subsys/net/l2/ethernet/ethernet.c

```diff
@@ -233,6 +233,7 @@ static enum net_verdict ethernet_recv(struct net_if *iface,
 	switch (type) {
 	case NET_ETH_PTYPE_IP:
 	case NET_ETH_PTYPE_ARP:
+	case NET_ETH_PTYPE_EAPOL:
```

Comment:
Shouldn't the family be `AF_UNSPEC` because this has nothing to do with IPv4 address family?

Reply:
Yes.